src/extensions/step_handlers/apt_handler/src/apt_handler.cpp

```diff
@@ -316,19 +330,22 @@ ADUC_Result AptHandlerImpl::Install(const ADUC_WorkflowData* workflowData)
         goto done;
     }
 
+    config = ADUC_ConfigInfo_GetInstance();
+    if (config == NULL)
+    {
+        Log_Error("Failed to get config instance.");
```

Comment:
string keeps logging string keeps repeating. Log in ADUC_ConfigInfo_GetInstance() or omit, since it should not happen

Reply:
line 186 and anywhere else calling ADUC_ConfigInfo_GetInstance

Reply:
It's better to log this at the caller so that we can see the caller function name in the log file, instead of seeing  [AUC_ConfigInfoGetIntance]